operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/rate/PeriodRateLimiter.java

```diff
@@ -6,38 +6,37 @@
 import java.util.Map;
 import java.util.Optional;
 
+import io.javaoperatorsdk.operator.api.config.AnnotationConfigurable;
 import io.javaoperatorsdk.operator.processing.event.ResourceID;
 
 /**
  * A Simple rate limiter that limits the number of permission for a time interval.
  */
-public class PeriodRateLimiter implements RateLimiter {
+public class PeriodRateLimiter
```

Comment:
Rate limiting is always for period, maybe name it as `LinearRateLimiter` (other one way is exponential, but actually usually not for this case how we use it here)